etc/profile-m-z/yt-dlp.profile

```diff
@@ -5,17 +5,65 @@ quiet
 # Persistent local customizations
 include yt-dlp.local
 # Persistent global definitions
-# added by included profile
-#include globals.local
+include globals.local
+
+# breaks when installed under ${HOME} via `pip install --user` (see #2833)
+ignore noexec ${HOME}
 
 noblacklist ${HOME}/.cache/yt-dlp
 noblacklist ${HOME}/.config/yt-dlp
 noblacklist ${HOME}/.config/yt-dlp.conf
 noblacklist ${HOME}/yt-dlp.conf
 noblacklist ${HOME}/yt-dlp.conf.txt
+noblacklist ${HOME}/.netrc
+noblacklist ${MUSIC}
+noblacklist ${VIDEOS}
+
+# Allow python (blacklisted by disable-interpreters.inc)
+include allow-python3.inc
+
+blacklist /tmp/.X11-unix
+blacklist ${RUNUSER}
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+ipc-namespace
+machine-id
+netfilter
+no3d
+nodvd
+nogroups
+noinput
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+seccomp
+seccomp.block-secondary
+tracelog
+
+private-bin env,ffmpeg,ffprobe,python*,yt-dlp
+private-cache
+private-dev
+private-etc @tls-ca,mime.types,yt-dlp.conf
+private-tmp
 
-private-bin ffprobe,yt-dlp
-private-etc yt-dlp.conf
+dbus-user none
+dbus-system none
 
-# Redirect
-include youtube-dl.profile
+#memory-deny-write-execute - breaks on Arch (see issue #1803)
```

Comment:
Is this still correct? I've been running with mdwe (on Arch Linux and Ubuntu LTS) in a local override for quite some time without apparent issues.

Reply:
@rusty-snake has it in [kyst](https://github.com/rusty-snake/kyst/blob/main/firejail/youtube-dl.local) as well. Enabling it seems well supported, let's do it!

Reply:
> i'll add them all at once

Okay :) Marking conversation as resolved.